react-basic-practice/src/components/TextArea/index.tsx

```diff
@@ -15,12 +15,14 @@ const Textarea: React.FC<CustomTextareaProps> = ({ errorMessage, ...rest }) => {
     <Box>
       <ChakraTextarea
         {...rest}
-        borderColor={errorMessage ? "brickRed" : rest.borderColor}
+        borderColor={errorMessage ? "brickRed" : "paleSky"}
+        isInvalid={false}
+        _focusVisible={{}}
```

Comment:
please re-check

Reply:
because I need to override chakraUI's default property